src/api/libcellml/specificationrules.h

```diff
@@ -27,9 +27,17 @@ namespace libcellml {
 enum class SpecificationRule
 {
     UNDEFINED,
+    DATA_REPR_IDENTIFIER_UNICODE,
+    DATA_REPR_IDENTIFIER_LATIN_ALPHANUM,
+    DATA_REPR_IDENTIFIER_ATLEAST1_ALPHANUM,
```

Comment:
For consistency and "cleanliness", replace with `DATA_REPR_IDENTIFIER_AT_LEAST_ONE_ALPHANUM`.